### ISSUE TITLE
Windows Accessibility issue and ReorderableListView crash

### ISSUE THREAD

**Issue by 0xh4n** [flutter/flutter#103808]
hi ,
my app crashes  (debug/release) should be  same as #100919 ,  #99539  ,  #100610 ,  it was in different machine i couldn't reproduced in my main machine . but  i finally found out why. and now i can reproduce it  in multiple machines  windows 10 / 11 . 
i also found out it can reproduce this too #98099

i tested this in 2 machines (windows10/11)  with both binaries compiled with my main dev machine and github actions  same result. 
also on 2 dev environment  (main machine and virtual machine )  
flutter 2.10  and 3.0  ,  Visual Studio 2019
## Steps to Reproduce

1. launch Window Magnifier (shortcut Win + Plus)  ->> **important: without this the app won't crash and will work perfectly**
2. `flutter run`  the code sample 
3. sometimes do step 2  multiple times until it crash 

**Expected results:** 
the app run and work fine

**Actual results:** 
the app will crash immediately most of the times .
if it doesn't crash ,  when you try to reorder tabs will get error in console   : 
ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree    #98099

release build will crash silently , debug build will get (abort/retry/cancel error dialog)

i was using TabView from fluent_ui   , but it uses ReorderableListView 
<details>
<summary>Code sample</summary>

```dart

import 'package:flutter/material.dart';

void main() {
  runApp(MaterialApp(
    home: Scaffold(
      body: ReorderableListView(
        onReorder: (_, __) {},
        children: [
          for (var i = 0; i < 10; i++) Text("Tab $i", key: GlobalKey())
        ],
      ),
    ),
  ));
}

```

</details>

<details>
  <summary>flutter doctor -v   from virtual machine</summary>

```
[√] Flutter (Channel stable, 3.0.0, on Microsoft Windows [Version 10.0.19044.1288], locale en-US)
    • Flutter version 3.0.0 at E:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ee4e09cce0 (5 days ago), 2022-05-09 16:45:18 -0700
    • Engine revision d1b9a6938a
    • Dart version 2.17.0
    • DevTools version 2.12.2

[X] Android toolchain - develop for Android devices
    X Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[X] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.14)
    • Visual Studio at e:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.32428.217
    • Windows 10 SDK version 10.0.19041.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] Connected device (2 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.1288]
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 92.0.902.67

[√] HTTP Host Availability
    • All required HTTP hosts are available

```
</details>

<details>
  <summary>flutter doctor -v  main dev machine</summary>

```

[√] Flutter (Channel stable, 2.10.5, on Microsoft Windows [Version 10.0.22000.652], locale en-US)
    • Flutter version 2.10.5 at E:\flutter2
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5464c5bac7 (4 weeks ago), 2022-04-18 09:55:37 -0700
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at E:\Android\Android-sdk\sdk
    • Platform android-31, build-tools 32.0.0
    • Java binary at: E:\Android\android-studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.13)
    • Visual Studio at E:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.32413.511
    • Windows 10 SDK version 10.0.22000.0

[√] Android Studio (version 2021.1)
    • Android Studio at E:\Android\android-studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[√] VS Code (version 1.67.1)
    • VS Code at C:\Users\User\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.40.0

[√] Connected device (1 available)
    • Windows (desktop) • windows • windows-x64 • Microsoft Windows [Version 10.0.22000.652]

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

</details>

https://user-images.githubusercontent.com/105427712/168428330-694eaff4-9bda-4ceb-b380-2dc7d6aa5b4c.mp4

**Comment by 0xh4n** [comment]
ok for now i'm not using  onReorder parameter in TabView ( fluent_ui package which seems uses ReorderableListView ) but that was not enough so i had to wrap TabView with ExcludeSemantics widget , since i don't need reorder functionality  this was solution for me .

**Comment by maheshmnj** [comment]
Hi @0xh4n, thanks for filing the issue, I am able to reproduce the crash on master channel while on the stable channel the console is being spammed with the accessibility error Which is the same as https://github.com/flutter/flutter/issues/98099. 

### Master


https://user-images.githubusercontent.com/31410839/168564271-32da697c-e649-4e08-a4cf-1f6882c51398.mp4



### Stable

https://user-images.githubusercontent.com/31410839/168563882-c895b284-55ab-441a-9a5b-f3116006f98d.mp4


<details>
<summary>logs (master)</summary>

```
PS C:\Users\mahesh\Desktop\keyboardissue> flutterm run -d windows
Running "flutter pub get" in keyboardissue...                    2,202ms
Launching lib\main.dart on Windows in debug mode...
Building Windows application...                                         
Syncing files to device Windows...                                 145ms

Flutter run key commands.
r Hot reload. 
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

 Running with sound null safety 

An Observatory debugger and profiler on Windows is available at:
http://127.0.0.1:53285/mMfQClXVUEY=/
The Flutter DevTools debugger and profiler on Windows is available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:53285/mMfQClXVUEY=/
Lost connection to device.
```
</details>

<details>
<summary>logs (stable)</summary>

```
Flutter run key commands.
r Hot reload. 
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application
running).
c Clear the screen
q Quit (terminate the application on the device).

 Running with sound null safety 

An Observatory debugger and profiler on Windows is availableat: http://127.0.0.1:52161/j99b64AkwTE=/
The Flutter DevTools debugger and profiler on Windows is
available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:52161/j99b64AkwTE=/
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 22 is not marked for destruction, would be 
reparented to 1
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTbe a new child
ree, error: Node 14 is already pending for creation, cannot 
be a new child

Application finished.
```
</details> 


<details>
<summary>flutter doctor -v (windows)</summary>

```
[√] Flutter (Channel stable, 3.0.0, on Microsoft Windows [Version
    10.0.19044.1645], locale en-IN)
    • Flutter version 3.0.0 at C:\flutter_sdk\stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ee4e09cce0 (6 days ago), 2022-05-09 16:45:18 -0700
    • Engine revision d1b9a6938a
    • Dart version 2.17.0
    • DevTools version 2.12.2

[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)    • Android SDK at C:\Users\mahesh\AppData\Local\Android\sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java      
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)    
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.4)  
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual
      Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.31727.386
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)    

[√] VS Code (version 1.64.2)
    • VS Code at C:\Users\mahesh\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.40.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 
      10.0.19044.1645]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 101.0.4951.54    • Edge (web)        • edge    • web-javascript • Microsoft Edge
      100.0.1185.50

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
```
[√] Flutter (Channel master, 3.1.0-0.0.pre.727, on Microsoft Windows [Version
    10.0.19044.1645], locale en-IN)
    • Flutter version 3.1.0-0.0.pre.727 at C:\flutter_sdk\master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 36f9a186bf (6 hours ago), 2022-05-15 14:34:06 -0400    
    • Engine revision 1b064fd1a3
    • Dart version 2.18.0 (build 2.18.0-116.0.dev)
    • DevTools version 2.13.1

[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)    • Android SDK at C:\Users\mahesh\AppData\Local\Android\sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java      
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)    
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.4)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual
      Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.31727.386
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)    

[√] VS Code (version 1.64.2)
    • VS Code at C:\Users\mahesh\AppData\Local\Programs\Microsoft VS Code       
    • Flutter extension version 3.40.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 
      10.0.19044.1645]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 101.0.4951.54    • Edge (web)        • edge    • web-javascript • Microsoft Edge
      100.0.1185.50

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by repentsinner** [comment]
Just a note that I've also seen random hard crashes with ReorderableListView in Flutter 3.0.0 on Windows 11. Also getting the accessibility errors in the console. I was able to repro them in a minimal app last night using the ReorderableListView [sample code](https://api.flutter.dev/flutter/material/ReorderableListView-class.html) (at the time it seemed like I needed to put it in a non-default route and then switch to that route to make it crash, but that could be a red herring since I am no longer able to repro), but as of this morning I can't get the sample code nor my app to reproduce the crash on the same machine. I understand that this isn't a lot of new information nor an MCVE, will continue trying to reproduce.

<details><summary>flutter doctor -v</summary>
[√] Flutter (Channel stable, 3.0.0, on Microsoft Windows [Version 10.0.22000.675], locale en-US)
    • Flutter version 3.0.0 at C:\Users\ritchie\Development\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ee4e09cce0 (2 weeks ago), 2022-05-09 16:45:18 -0700
    • Engine revision d1b9a6938a
    • Dart version 2.17.0
    • DevTools version 2.12.2

[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at C:\Users\ritchie\AppData\Local\Android\sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.1.6)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.1.32421.90
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)

[√] VS Code (version 1.67.2)
    • VS Code at C:\Users\ritchie\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.40.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.675]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 101.0.4951.67
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 101.0.1210.39

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
</details>

**Comment by 0xNF** [comment]
I created a new flutter project, using only the code contained the ReorderableListView sample app code, called it "reorderablecrash", then opened up the generated `build/windows/reorderablecrash.sln` file. I set the startup project to be `reorderablecrash`. I get the following crash info pretty reliably on startup:

```
Exception thrown at 0x00007FFDD1995CC7 (flutter_windows.dll) in reorderablecrash.exe: 0xC0000005: Access violation reading location 0x00000233E5FF9FF0.
```
I'll add any more info to this post if and when I find it.

**Comment by 0xNF** [comment]
@maheshmnj 
I think I've done just about as much work on this as I can -- I don't have the Flutter engine knowledge or the C++ expertise to uncover a better fix than the one I've posted. Ideally, I'd look at tracing the specifics of the ReorderableWrap widget implementation and trying to figure out why the iters aren't the right lengths, but I think that's pretty far out of my wheelhouse. 

I can push this `continue` change to my own fork and make a PR for it, but this just treats the symptoms and I don't have a good fix for the core problem.

**Comment by cbracken** [comment]
Given the frequency of reports, I've bumped the priority on this. @0xNF and @0xh4n thank you for the detailed comments.

**Comment by AniketMalik** [comment]
Do we have a solution ?

**Comment by loic-sharma** [comment]
@cbracken I was able to reproduce this issue on my computer, I can investigate this if you'd like.

**Comment by cbracken** [comment]
Awesome -- if you're able to repro this would be great to get fixed.

**Comment by 0xNF** [comment]
With all due respect, the fact that the flutter team can't reproduce this bug, after dozens of independent reports and all the effort out into this by volunteers, is pretty disheartening. I don't get the sense that you guys care about desktop at all except as a distant afterthought.

**Comment by loic-sharma** [comment]
Thanks for the feedback @0xNF, we appreciate all the effort you put in to provide excellent details and a repro app. This is incredibly useful. Several folks joined the Flutter desktop team recently, so we should be able to address these kinds of issues faster going forward.

**Comment by loic-sharma** [comment]
Hello everyone, this is now fixed in the `master` channel. You can try this using:

```
flutter channel master
flutter upgrade
```

Please let me know if you can still reproduce this issue while on the `master` channel!

**Comment by 0xh4n** [comment]
hello , from my test looks like that fixed the crash  .
**note** that switching to master channel was not enough for me, i had to manually change engine revision that has this fix to test ( i used: ead1348803aacfaf6b5bf474f4a8b9a5eac625c4 ). 


<details>

  <summary>flutter doctor -v </summary>

```
[✓] Flutter (Channel master, 3.1.0-0.0.pre.2592, on Microsoft Windows [Version 10.0.22000.832], locale en-US)
    • Flutter version 3.1.0-0.0.pre.2592 on channel master at E:\flutter\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 069f504297 (12 hours ago), 2022-08-30 18:20:09 -0700
    • Engine revision ead1348803
    • Dart version 2.19.0 (build 2.19.0-152.0.dev)
    • DevTools version 2.16.0

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at E:\Android\Android-sdk\
    • Platform android-33, build-tools 33.0.0
    • Java binary at: E:\Android\android-studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.13)
    • Visual Studio at E:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.32413.511
    • Windows 10 SDK version 10.0.22000.0

[✓] Android Studio (version 2021.2)
    • Android Studio at E:\Android\android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+7-b1504.28-7817840)

[✓] VS Code (version 1.70.2)
    • VS Code at C:\Users\User\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.44.0

[✓] Connected device (1 available)
    • Windows (desktop) • windows • windows-x64 • Microsoft Windows [Version 10.0.22000.832]

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

</details>

**Comment by loic-sharma** [comment]
Thanks for verifying @0xh4n!

> note that switching to master channel was not enough for me, i had to manually change engine revision to one that has this fix to test ( i used: ead1348803aacfaf6b5bf474f4a8b9a5eac625c4 ).

That's my mistake, I commented before the engine was rolled into the `master` channel 🤦
